thorlcr/thorutil/thormisc.hpp

```diff
@@ -69,6 +71,9 @@
 #define THOROPT_LKJOIN_HASHJOINFAILOVER "lkjoin_hashjoinfailover" // Force SMART to failover to hash join (for testing only)                     (default = false)
 #define THOROPT_MAX_KERNLOG           "max_kern_level"          // Max kernel logging level, to push to workunit, -1 to disable                  (default = 3)
 #define THOROPT_COMP_FORCELZW         "forceLZW"                // Forces file compression to use LZW                                            (default = false)
+#define THOROPT_COMP_FORCEFLZ         "forceFLZ"                // Forces file compression to use FLZ                                            (default = false)
+#define THOROPT_COMP_FORCELZ4         "forceLZ4"                // Forces file compression to use LZ4                                            (default = false)
+#define THOROPT_COMP_FORCEDEN         "forceDEN"                // Forces file compression to use Density                                        (default = false)
```

Comment:
Not convinced any of these options are worth it any more.
The forceLZW option was to override the default of using RowDiff compression for fixed size records..

If the std. default (LZ4) is now as fast or faster than rowdiff with reasonable compression rates (is that true?), then I don't think we should special case the fixed length case and we can get rid of these forceXXX options too.